models/url.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+import (
+	"time"
+
+	"github.com/uptrace/bun"
+)
+
+type Tinyurl struct {
+	bun.BaseModel `bun:"table:tiny_url"`
+
+	Id        int64     `bun:"id,pk,autoincrement"`
+	OrgUrl    string    `bun:"org_url,notnull"`
+	ShortUrl  string    `bun:"short_url,unique,notnull"`
+	Comment   string    `bun:"comment"`
+	UserId    int       `bun:"user_id"`
```

Comment:
- add a foreign key constraint here  like done in the migrations file

Reply:
done